DataConnectors/Qualys VM/profile.ps1

```diff
@@ -0,0 +1,20 @@
+# Azure Functions profile.ps1
```

Comment:
What's changed here? This is same as the profile.ps1 file that's part of PR#809 that we merged recently - file at  https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Qualys%20VM/profile.ps1

Reply:
no changes here, these are just generic/default files created with every function app